travel/src/main/java/com/tx/travel/payload/response/UserInfoResponse.java

```diff
@@ -6,15 +6,76 @@ public class UserInfoResponse {
   private Long id;
   private String username;
   private String email;
+
+  private String passport_no;
+  private String id_no;
+  private String name;
+  private String surname;
+  private Long cost_center_id;
+
   private List<String> roles;
 
-  public UserInfoResponse(Long id, String username, String email, List<String> roles) {
+  public UserInfoResponse(
+      Long id,
+      String username,
+      String email,
+      String passport_no,
+      String id_no,
+      String name,
+      String surname,
+      Long cost_center_id,
+      List<String> roles) {
     this.id = id;
     this.username = username;
     this.email = email;
+    this.passport_no = passport_no;
+    this.id_no = id_no;
+    this.name = name;
+    this.surname = surname;
+    this.cost_center_id = cost_center_id;
     this.roles = roles;
   }
 
+  public String getPassport_no() {
```

Comment:
camel case in method and variable names please! - apply to all methods and variables in all classes